test/datasets_utils.py

```diff
@@ -748,6 +751,49 @@ def size(idx: int) -> Tuple[int, int, int]:
     ]
 
 
+def shape_test_for_stereo_gt_w_mask(
```

Comment:
To avoid repeating code too much I'd suggest merging `shape_test_for_stereo_gt_w_mask` and `shape_test_for_stereo_gt_no_mask` and `shape_test_for_stereo_no_gt` together, possibly with new `no_gt` and `no_mask` boolean parameters